pkgs/by-name/ct/ctrld/package.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nixosTests
+}:
+
+buildGoModule rec {
+  pname = "ctrld";
+  version = "1.3.3";
+
+  src = fetchFromGitHub {
+    owner = "Control-D-Inc";
+    repo = "ctrld";
+    rev = "v${version}";
+    hash = "sha256-vFXCBonJCZWFy+sKXDQ55ykFb67Sj1GdfuMtKIf4+5I=";
+  };
+
+  vendorHash = "sha256-il3jA4Dtfs1GMyL3IEgmzRDOIsKViovBqhtsvzwgafQ=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X=main.version=${version}"
+    "-X=main.commit=${src.rev}"
+  ];
+
```

Seems like this got removed

```suggestion

  # requires network
  doCheck = false;

```

---

Done.